src/shim/telemetryClient.ts

```diff
@@ -436,4 +712,16 @@ export class TelemetryClient {
         const record = this._telemetryToLogRecord(telemetry, baseType, baseData);
         return record;
     }
+
+    public trackNodeHttpRequestSync(telemetry: Contracts.NodeHttpRequestTelemetry) {
```

Comment:
If customer try to directly NodeClient your code will fail, these methods should be in a class called NodeClient